application/controllers/Qso.php

```diff
@@ -29,7 +29,8 @@ public function index()
         if(!$this->user_model->authorize(2)) { $this->session->set_flashdata('notice', 'You\'re not allowed to do that!'); redirect('dashboard'); }
 
 		$data['active_station_profile'] = $this->stations->find_active();
-        
+		$data['qslmsg'] = $this->stations->get_station_info($data['active_station_profile'],'eqsl_defaultqslmsg', '');
```

Comment:
Why using (new) get_station_info where a function called "profile" alrteady exists?

Reply:
profile return all data, i think to create a function like get_station_power(), but more generic. 
ok for use profile() if you prefered